packages/shared/src/components/widgets/PostUsersHighlights.tsx

```diff
@@ -78,15 +79,22 @@ const Image = (props: SourceAuthorProps) => {
 const UserHighlight = (props: SourceAuthorProps) => {
   const { id, handle, name, username, permalink, userType = 'source' } = props;
   const Icon = getUserIcon(userType);
-  const LinkWrapper = userType === 'source' ? React.Fragment : ProfileTooltip;
+  const userTypeNotSource = userType === 'source';
```

Comment:
I think there was a typo.
```suggestion
  const userTypeNotSource = userType !== 'source';
```

Reply:
Ah yes my bad was testing it and accidently pushed the wrong one 🙈